structuralcodes/codes/mc2010/_concrete_torsion.py

```diff
@@ -0,0 +1,172 @@
+"Covers torsion in Model code 2010, 7.3.4"
+
+from math import pi, tan, sin, cos
+from ._concrete_shear import epsilon_x, v_rd_max, eta_fc
+
+
+def v_ed_ti(t_ed: float, a_k: float, z_i: float):
+    """Shear force due to torsion
+
+    fib Model Code 2010, eq. (7.3-53)
+
+    Args:
+        t_ed: The acting torsion force in the cross section in Nmm
+        z_i: Can be found in figure 7.3-18
+        a_k: Can be found in figure 7.3-18
+
+    Returns:
+        The shear force that will ocurre due to torsion force."""
+    return t_ed * z_i / (2 * a_k)
+
+
+def t_rd_max(
+    f_ck: float,
+    d_k: float,
+    a_k: float,
+    theta: float,
+    approx_lvl: int,
+    z: float,
+    E_s: float,
+    As: float,
+    loads: dict,
+    gamma_c: float = 1.5,
+) -> float:
+    """The maximum allowed torsion allowed
+
+    fib Model Code 2010, eq. (7.3-56)
+
+    Args:
+        f_ck (float): Characteristic strength in MPa
+        gamma_c (float): Concrete safety factor
+        d_k (float): Is the diameter in the smalest circel in the cross section
+        a_k: Can be found in figure 7.3-18
+        theta (float): Inclitaniton of the compression stressfield in degrees
+        approx_lvl (int): Approximation method for cocrete with reinforcement
```

Comment:
Please revisit the levels of approximation as in the section on transverse shear. This should be reflected in the if-block below.